styles.css

```diff
@@ -290,3 +290,138 @@
     line-height: 40px;
     margin-bottom: 45px;
 }
+
+/* Chefs Section*/
+
+#chefs-section {
+    text-align: center;
```

Comment:
You also added a chefs section to the testimonials section. Please fix it.

Reply:
Good catch. I'll fix it